ocs_ci/helpers/performance_lib.py

```
@@ -625,149 +627,172 @@ def get_pvc_provision_times(interface, pvc_name, start_time, time_type="all", op
-                                if results[name]["csi_delete"]["end"] is None:
-                                    results[name]["csi_delete"]["end"] = (
-                                        extruct_timestamp_from_log(line)
-                                    )
-                                    results[name]["csi_delete"]["time"] = (
-                                        calculate_operation_time(
-                                            name, results[name]["csi_delete"]
-                                        )
+
+                        if "GRPC call" in line and "DeleteVolume" in line:
+
+                            if results[name]["csi_delete"]["start"] is None:
```

Ruff (0.15.17)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)